config/vufind/RecordTabs.ini

```diff
@@ -39,6 +39,13 @@ tabs[Preview] = preview
 tabs[Details] = StaffViewArray
 defaultTab = null
 
+[VuFind\RecordDriver\ProQuestFSG]
```

Comment:
I've removed the Similar Items tab as @sturkel89 noted
> Similar Items tab error

I also removed the Holdings tab.  It wasn't broken, but it added no useful links that I can find beyond what's already in the core metadata fields.  I think there's a larger conversation to have about where these links should be, but there didn't seem to be a point in displaying them twice.  I am though questioning that.